jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -261,6 +262,39 @@ class EndpointK @JvmOverloads constructor(
         return _transceiver.isReceivingVideo()
     }
 
+    /**
+     * Update media direction of {@link ChannelShim}s associated
```

Comment:
Is this the right syntax for KDoc?

Reply:
I think this style is still supported, but I'll change it.